src/core/webidl.js

```diff
@@ -326,9 +330,8 @@ function renderWebIDL(idlElement, index) {
   }
   // we add "idl" as the canonical match, so both "webidl" and "idl" work
   idlElement.classList.add("def", "idl");
-  const html = webidl2.write(parse, { templates });
-  const render = html.bind(idlElement);
-  render`${html}`;
+  const highlights = webidl2.write(parse, { templates });
+  html.bind(idlElement)`${highlights}`;
```

Comment:
There should be a native function to do this work, something like `idlElement.replaceChildren(...newChildren)`

Reply:
Worth chatting to Annevk about it.